charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -31,6 +41,16 @@ sidecars:
       pullPolicy: IfNotPresent
       repository: k8s.gcr.io/sig-storage/csi-attacher
       tag: "v3.4.0"
+    # Tune leader lease election for csi-attacher.
+    # Leader election is on by default.
+    leaderElection:
+      enabled: true
```

Comment:
The reason I defaulted to true is because this was previously hardcoded to "true" in the helm template.

Thus if we default to false here then it breaks the default that most users of this template are relying on today.

Reply:
I see, you're totally right - let's leave it as `true` in that case.